recipes_source/mobile_perf.rst

```diff
@@ -221,3 +221,45 @@ Now we are ready to benchmark your model:
   Main run finished. Microseconds per iter: 121318. Iters per second: 8.24281
 
 
+iOS
+-------
+
+For iOS, we'll be using our `TestApp<https://github.com/pytorch/pytorch/tree/master/ios/TestApp>` as the benchmarking tool. 
```

Comment:
Syntax: Add an underscore at the end and a space after TestAPP in order for the hyperlink to render. 

```text
`TestApp <https://github.com/pytorch/pytorch/tree/master/ios/TestApp>`_ 
```